### PR TITLE
Fix 109 heatmap bug

### PR DESCRIPTION
**Pull request by SejoB**
- this widget should display data only if have more then two different points

### CODE REVIEW

File: src/components/templates/WidgetsTemplate.tsx [hunk 1]
```diff
@@ -2,43 +2,46 @@ import React, {FunctionComponent, useEffect} from 'react'
...
 
 const getWidgetByType = (widget: any) => {
   let widgetComponent;
-
   switch (widget.name) {
     case 'most_severe_accidents': {
       widgetComponent = <LocationMap data={widget.data} center={{lat: 32.0853, lng: 34.7818}}/>;
       break;
     }
     case 'accidents_heat_map': {
```
Comment (danielsh28) [inline src/components/templates/WidgetsTemplate.tsx:27]: is it need to be  `accidents_heat_map'` or `'most_severe_accidents_heatmap'`?

Comment (danielsh28) [inline src/components/templates/WidgetsTemplate.tsx:27]: is it need to be  `accidents_heat_map'` or `'most_severe_accidents_heatmap'`?

File: src/index.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
-import React from 'react';
```
Comment (danielsh28) [inline src/index.tsx:1]: use single quotes instead of double quotes

File: src/components/templates/WidgetsTemplate.tsx (hunk 1 above)
Comment (SejoB) [reply src/components/templates/WidgetsTemplate.tsx:27]: accidents_heat_map - it's what coming from the server
most_severe_accidents_heatmap - this one only on mock

File: src/index.tsx (hunk 2 above)
Comment (SejoB) [reply src/index.tsx:1]: fixed

File: .gitignore [hunk 3]
```diff
@@ -22,3 +22,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.vscode/settings.json
```
Comment (yuvalbl) [inline .gitignore:25]: should ignore `vscode` directory entirely

File: src/components/templates/WidgetsTemplate.tsx (hunk 1 above)
Comment (yuvalbl) [reply src/components/templates/WidgetsTemplate.tsx:27]: we should change mock
`most_severe_accidents_heatmap` => `most_severe_accidents_heatmap`

**Pull request merged**